mlir/utils/performance/perfRunner.py

```diff
@@ -23,11 +24,20 @@
 from perfCommonUtils import Operation, GEMMLibrary
 
 # global variables.
-ROCPROF = '/opt/rocm/bin/rocprof'
-MIOPENDRIVER = '/opt/rocm/bin/MIOpenDriver'
-BENCHMARKING_RESULT_FILE_NAME = 'results.stats.csv'
-BENCHMARKING_METRICS_FILE_NAME = 'results.csv'
+BENCHMARKINGV1_RESULT_FILE_NAME = 'results.stats.csv'
```

Comment:
Of course things move around for rocprofv2, and there's no way I found to make them the same.  In particular, that "pmc_1" directory inserts itself because of either --kernel-trace or the stats in -i, I forget which.